src/Models/Social.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace TruckersMP\Models;
+
+class Social
+{
+    /**
+     * @var string|null
+     */
+    protected $twitter;
+
+    /**
+     * @var string|null
+     */
+    protected $facebook;
+
+    /**
+     * @var string|null
+     */
+    protected $plays;
+
+    /**
+     * @var string|null
+     */
+    protected $discord;
+
+    /**
+     * @var string|null
+     */
+    protected $youtube;
+
+    /**
+     * Create a new Social instance.
+     *
+     * @param array $social
+     */
+    public function __construct(array $social)
```

Comment:
Shouldn't the construct rather accept parameters of social sites instead of an array? Like the model `Game`.